control-plane/roles/monitoring/README.md

```diff
@@ -0,0 +1,37 @@
+# monitoring
+
+This role is designed to set up monitoring for a Linux-based system using Ansible. 
+The role includes tasks to install and configure the following monitoring tools:
+-    Prometheus
+-     Grafana
+-    Node Exporter
+-    Alertmanager
+-    Thanos
```

Comment:
Indentions are to deep, also leads to wrong markdown rendering. See:

-    Prometheus
-     Grafana
-    Node Exporter
-    Alertmanager
-    Thanos

And

- Prometheus
- Grafana
- Node Exporter
- Alertmanager
- Thanos

Reply:
https://github.com/metal-stack/metal-roles/commit/54f9352336151a7d713da3516cec2cf89ce7fd3c